queries/nix.scm

```diff
@@ -337,14 +277,12 @@
 ;;!  xxxxxx
 ;;!  --------
 (apply_expression
-  function: (variable_expression
-    name: (identifier) @functionCallee
-  )
   argument: (_) @argumentOrParameter
-) @functionCall @_.domain
+)
 
-(apply_expression
-  argument: (_) @argumentOrParameter
+(
+  (apply_expression) @argumentOrParameter.iteration
+  (#not-parent-type? @argumentOrParameter.iteration apply_expression)
 )
```

Comment:
@fidgetingbits this isn't entirely related, but note that we can simply define iteration scope for arg like this, and then "first arg" etc should just work